include/gfx/Texture.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "gfx/texture/Texture.h"
+#include "gfx/texture/TextureFilterMode.h"
+#include "gfx/texture/TextureWrapMode.h"
+#include "texture/TextureManager.h"
+
+namespace arch::gfx {
+
+using namespace arch::gfx::texture;
```

Comment:
Should everything from `gfx::texture` be visible from `arch`?

Reply:
no, will be visible in `arch::gfx` tho